content/services/development.md

```diff
@@ -9,9 +9,180 @@ action:
   text: See work samples »
   url: /work/
 summary: |
-  **We integrate design and development in an agile process**,
-  so you only need to hire one team --
-  completely focused on achieving your goals
-  with smooth and efficient progress
-  from concept to launch and beyond.
+  **We integrate design and development in an agile process**, so you only need to hire one team – completely focused on achieving your goals with smooth and efficient progress from concept to launch and beyond. Starting from user needs, we collaborate with you to deliver usable features quickly and efficiently.
+
+  We provide expertise across a range of technologies, including **Python/Django**, accessible **HTML**, modern **Sass/CSS**, **Vue**, **Node**, **Backbone/Marionette**, and more.
 ---
+
+{% import 'embed.macros.njk' as embed %}
+{% import 'quotes.macros.njk' as quotes %}
+{% import 'layout.macros.njk' as layout %}
+
+{{ quotes.find(
+  collections.all,
+  slugs=['realistic', 'curious']
+) }}
+
+{{ layout.title('Services') }}
+
+{% call embed.media_block(
+  media=embed.screen(
+    content=embed.img(
+      src='pages/trig-phone.jpg',
+      alt='trig race screen',
+      width=744 ,
+      height=1611,
+      sizes='media'
+    )
+  )
+) %}
+
+{% call layout.grid('narrow-columns') %}
+
+{% call layout.block('item') %}
+
```

Comment:
@mirisuzanne @jgerigmeyer I wasn't sure if line break matter as much here as they were in RST blog. I am not sure if we have a preferred amount of space between these calls and the end ones (there are three in a row after the second column of content).

Reply:
@stacyk Generally, spacing shouldn't matter too much – but it can have some impact on how the file is read as markdown. We don't have specific conventions, I tend to base it on what blocks are related or what seems most readable.